lib/address-derivation-discovery/lib/Cardano/Wallet/Address/Derivation/Icarus.hs

```diff
@@ -282,11 +284,7 @@ generateKeyFromHardwareLedger (SomeMnemonic mw) (Passphrase pwd) = unsafeFromRig
 
     -- As described in [BIP 0039 - From Mnemonic to Seed](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed)
     pbkdf2HmacSha512 :: ByteString -> ByteString
-    pbkdf2HmacSha512 bytes = PBKDF2.generate
-        (PBKDF2.prfHMAC SHA512)
-        (PBKDF2.Parameters 2048 64)
-        bytes
-        ("mnemonic" :: ByteString)
+    pbkdf2HmacSha512 = generateKeyIcarus
```

Comment:
I would like to keep the specification of the numerical values in `Cardano.*`. See other comment.

Reply:
sounds reasonable. done